hack/test-cmd.sh

```diff
@@ -112,7 +112,7 @@ do
     SERVER_HOSTNAME_LIST="${SERVER_HOSTNAME_LIST},${IP_ADDRESS}"
 done <<< "${ALL_IP_ADDRESSES}"
 
-openshift admin create-master-certs \
+openshift admin ca create-master-certs \
```

Comment:
Be sure to add an equivalent line back using `openshift admin create-master-certs`.  If we have to be backwards compatible, we need to leave the old command in there to be sure we never break it.